apps/demos/Demos/DropDownButton/Overview/jQuery/index.js

```diff
@@ -20,11 +20,36 @@ $(() => {
     onItemClick(e) {
       DevExpress.ui.notify(e.itemData.name, 'success', 600);
     },
-    text: 'Sandra Johnson',
-    icon: '../../../../images/gym/coach-woman.png',
+    text: 'Olivia Peyton',
     displayExpr: 'name',
     keyExpr: 'id',
     useSelectMode: false,
+    template(data, element) {
+      const $imageWrapper = $("<div>")
+        .addClass('button-img-container').appendTo(element);
+
+      $("<div>")
+        .addClass('button-img-indicator').appendTo($imageWrapper);
+
+      $("<img>")
+        .addClass('button-img')
+        .attr({
+          src: '../../../../images/employees/51.png',
+          alt: 'employee',
+        }).appendTo($imageWrapper);
+
+      const $wrapper = $("<div>")
```

Comment:
```suggestion
      const $wrapper = $("<div>")
```

Same with `wrapper`. I would rename it to `textContainer`.